.gitignore

```diff
@@ -3,5 +3,7 @@ _site/
 .sass-cache/
 .jekyll-metadata
 Gemfile.lock
+_config.yml.bak
 .vendor/
 .bundle/
+id_ed25519
```

Comment:
this is the ssh identity file (deploy key) which allows travis to push to this repository, to prevent any potential problems we just exclude it from being tracked by git